Multisig/UI/Settings/ImportOwnerKey/PrivateKeyController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  PrivateKeyController.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 25.01.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class PrivateKeyController {
```

Comment:
Do we need to have this separated in controller? 
Isn't app settings enough?

If we want to go with this, how about moving remove key here also ?

Reply:
Yes, placing removing key logic is a good idea